Assets/_Game/Scripts/UI/Mushroom/MushroomUIController.cs

```diff
@@ -0,0 +1,78 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class MushroomUIController : MonoBehaviour
+{
+    
+    [SerializeField] Sprite yellowMushroom, redMushroom, purpleMushroom, blueMushroom;
+
+    [SerializeField]
+    private float remainingTime;
+
+    private SpriteRenderer sr;
+
+    private void Awake()
+    {
+        sr = GetComponent<SpriteRenderer>();
+    }
+    private void OnEnable()
+    {
+        EventManager.onCandleColorChanged += EventManager_onCandleColorChanged;
+    }
+
+    private void OnDisable()
+    {
+        EventManager.onCandleColorChanged -= EventManager_onCandleColorChanged;
+    }
+
+    private void Update()
```

Comment:
Do we need this update method?

Reply:
Doesn't look like it since the meat within update is commented out. I commented out the method itself in this commit -> 36356a8